usr/iscsiadm.c

```diff
@@ -699,21 +708,22 @@ static void print_nodes_tree(struct iscsi_node **nodes, uint32_t node_count)
 		if ((pre_node == NULL) ||
 		    (strcmp(iscsi_node_target_name_get(cur_node),
 			    iscsi_node_target_name_get(pre_node)) != 0))
-			printf("Target: %s\n",
+			printf("%sTarget: %s\n", prefix,
 			       iscsi_node_target_name_get(cur_node));
 		if ((pre_node == NULL) ||
 		    (strcmp(iscsi_node_conn_address_get(cur_node),
 			    iscsi_node_conn_address_get(pre_node)) != 0) ||
 		    (iscsi_node_conn_port_get(cur_node) !=
 		     iscsi_node_conn_port_get(pre_node)))
-			printf("\tPortal: %s,%d\n",
+			printf("%s\tPortal: %s,%d\n", prefix,
 			       iscsi_node_portal_get(cur_node),
 			       iscsi_node_tpgt_get(cur_node));
 		if ((pre_node == NULL) ||
 		    (strcmp(iscsi_node_iface_name_get(cur_node),
 			    iscsi_node_iface_name_get(pre_node)) != 0))
-			printf("\t\tIface Name: %s\n",
-			       iscsi_node_iface_name_get(cur_node));
+			if (! iface_mode)
```

Comment:
You generally do not put a space between the bang and the variable name.

Reply:
Sure. Will fix that.